### ISSUE TITLE
Diarization does not working

### ISSUE THREAD

**Issue by Rubenacp** [jhj0517/Whisper-WebUI#373]
**Which OS are you using?**
 - OS:  Windows 11
The problem is that when I active Diarization does not recognize the Speakers|. Instad of that appears None|
I use a RTX 2060 12 VRAM . 
Cuda 12.2 in CUDA_PATH but in CMD: nvidia-smi  the cuda version is 12.6 
python 11.5
I already installed it wit cu124  and cu121 in requirements.txt  and manually accept their terms in huggingface  (segmentation-3.0 and speaker-diarization-3.1), but I didn't see the models being downloaded. If I download them manually, in which folder do I have to put them?

--------------------
INFO:speechbrain.utils.quirks:Applied quirks (see `speechbrain.utils.quirks`): [disable_jit_profiling, allow_tf32]
INFO:speechbrain.utils.quirks:Excluded quirks specified by the `SB_DISABLE_QUIRKS` environment (comma-separated list): []
F:\Whisper-WebUI\venv\Lib\site-packages\pyannote\audio\utils\reproducibility.py:74: ReproducibilityWarning: TensorFloat-32 (TF32) has been disabled as it might lead to reproducibility issues and lower accuracy.
It can be re-enabled by calling
   >>> import torch
   >>> torch.backends.cuda.matmul.allow_tf32 = True
   >>> torch.backends.cudnn.allow_tf32 = True
See https://github.com/pyannote/pyannote-audio/issues/1370 for more details.

  warnings.warn(
F:\Whisper-WebUI\venv\Lib\site-packages\pyannote\audio\models\blocks\pooling.py:104: UserWarning: std(): degrees of freedom is <= 0. Correction should be strictly less than the reduction factor (input numel divided by output numel). (Triggered internally at C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\ReduceOps.cpp:1823.)
  std = sequences.std(dim=-1, correction=1)
F:\Whisper-WebUI\venv\Lib\site-packages\torch\cuda\memory.py:365: FutureWarning: torch.cuda.reset_max_memory_allocated now calls torch.cuda.reset_peak_memory_stats, which resets /all/ peak memory stats.
  warnings.warn(

-------------
Done in 8 seconds! Subtitle file is in the outputs folder.

None|Do you prefer to take photos of yourself or other things?
None|Of other things.
None|It's kind of hard to take pictures of yourself unless it's a selfie and that's limited.
None|So pictures of other things, mostly.
None|Now let's talk about animals.
None|Do you like animals?
None|I love animals.
None|Land animals, friendly, so to speak, animals, not insects and not anything from the ocean.
None|What is your favorite animal?
None|Probably a dog.
None|Dogs, for obvious reasons, you can keep them as pets.
None|In terms of like non-domesticated animals, probably jaguars.
None|Do you have any animals in your home as pets?
None|Yes, I have two dogs.
None|They're both from the shelter, the animal shelter, the rescues.
None|I've always had pets.
None|Our family really likes keeping animals around.

Thank you!

**Comment by jhj0517** [comment]
Hi @Rubenacp. Thanks for pointing that out!
Some recent patches were problematic, fixed this in #374. 

It should work fine now!

**Comment by Rubenacp** [comment]
Many thanks, it's working now
    El sábado, 2 de noviembre de 2024, 08:38:00 a.m. GMT+1, jhj0517 ***@***.***> escribió:  
 
 


Closed #373 as completed.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you were mentioned.Message ID: ***@***.***>
  

|  | Libre de virus.www.avast.com |
